### ISSUE TITLE
Not triggering on links anymore.

### ISSUE THREAD

**Issue by agzam** [awth13/org-appear#46]
For some reason it stopped working for links and I can't figure out why. What's the best method of debugging this?

I'm trying to eval `(org-appear-manual-start)` while cursor at the link, it reports returning `t`, but nothing is happening.

I tried funneling things through the Profiler (simply moving cursor in and out, around the square brackets of a link), it shows `org-appear--post-cmd`, but perhaps, it doesn't seem to trigger some preliminary function. Is that right?

    33,282   1% - org-appear--post-cmd
    32,258   1%  - org-appear--current-elem
    32,258   1%   - org-element-context
    19,648   0%    - org-element-at-point
    19,648   0%     - org-element--parse-to
     8,384   0%        org-element-headline-parser
     7,600   0%    - org-element--object-lex
     7,600   0%     - org-element-link-parser
     1,024   0%        org-link-unescape

Testing it the same way with "bold" elements appears to run `org-appear--show`:

    10,704   0%   - org-appear--show-with-lock
    10,704   0%    - org-appear--current-elem
    10,704   0%     - org-element-context
    10,704   0%      - org-element--object-lex
    10,704   0%       - org-element-bold-parser
    10,704   0%        - org-element--parse-generic-emphasis
    10,704   0%         - rx-to-string
    10,704   0%          - rx--translate
    10,704   0%           - rx--translate-form
    10,704   0%            - rx--translate-seq
    10,704   0%             - rx--translate
    10,704   0%              - rx--translate-form
    10,452   0%               + rx--translate-or
    252      0%               - rx--translate-not
    252      0%                  rx--generate-alt
 
And it works with bold elements as before. Why it's no longer working for links? Can't figure it out.

**Comment by awth13** [comment]
Hi, @agzam! Thank you for posting the issue!

What is your Org and Emacs version? I suspect this issue might be related to #5 -- you can check if your Org version has `org-fold` merged by evaluating `(featurep 'org-fold)`.

**Comment by artelse** [comment]
Am having the same problem with links not expanding. I run Emacs 29 master and org 9.5.4 and `org-fold` is present (of course).

**Comment by agzam** [comment]
Mine is Org mode version 9.6 (9.6-??-7c51329), running on GNU Emacs 29.0.50 (build 2, x86_64-apple-darwin21.5.0, NS appkit-2113.50 Version 12.4 (Build 21F79)) of 2022-06-05

**Comment by awth13** [comment]
Thank you again, @agzam and @artelse, for the input! The issue is now fixed.

**Comment by artelse** [comment]
Tested, fixed, thx!

**Comment by agzam** [comment]
@awth13 Thank you for fixing it promptly. I discovered 'org-appear' a long time ago, thought: "wow, this is cool", added it to my config and quickly forgot about it.

The most efficient software is like a killer move in the martial arts - the deadliest blows cannot be seen with a naked eye, things happen in an instance - there's nothing cinematic or cool about the fight.

That's what I felt using 'org-appear'. It has become part of my daily workflow. It's been great and worked almost undetected. Until one day it stopped working. And then I felt miserable. I realized how much I've become accustomed to the tiny feature of gracefully hiding and revealing things. Thank you for this gift. Thank you for giving me (and many others) the power of being an Org-mode ninja.
